components/dropdown-menu/index.js

```diff
@@ -149,9 +151,33 @@ export class DropdownMenu extends Component {
 		}
 	}
 
+	calculateMenuPosition() {
+		const { toggle } = this.nodes;
+		if ( toggle ) {
```

Comment:
We could avoid nesting the rest of the function by making this an early return instead:

```js
if ( ! toggle ) {
	return;
}
```

Reply:
Done